plugins/torch/plugin_test/test_torch.py

```diff
@@ -13,7 +13,7 @@
 
 @pytest.fixture
 def db() -> Iterator[Datalayer]:
-    db = superduper("mongomock://test_db")
+    db = superduper("mongomock://test_db", force_apply=True)
```

Comment:
u can even add it to config file so that u don't have to do this.

Reply:
For the plugin, instead of maintaining a separate configuration file, it would be more convenient for testing to define the database directly. WDYT?